doc/hooks/versionCheckHook.section.md

```diff
@@ -16,20 +22,67 @@ stdenv.mkDerivation (finalAttrs: {
+| Rebuild after modification             | The hook rebuilds the package when the hook is modified, since it is a phase running during the build time of the package.                                    | The tester does not require rebuilding the package, since it is a derivation.                                                                                                                               |
+| Overhead during package building       | Negligible. Although running during the build time of the package, the hook is lean and executes few commands.                                                | Zero, since the tester is a derivation that runs after package building.                                                                                                                                    |
+| Content-addressed-derivation awareness | Since it runs during the building, the hook does not deal with failures happening after building, like rewritings that happen post installation.              | Since it runs after the building, the tester detects failures at this time.                                                                                                                                 |
+| Execution by OfBorg CI tool            | Yes, since the hook is a phase running during the build time of the package.                                                                                  | OfBorg does run the tester for _directly affected packages only_; transitive dependencies are ignored by default, requiring extra commands like `@ofborg build dependency1.tests dependency2.tests ...`.    |
+| Execution by `nixpkgs-review` tool     | Yes, since the hook is a phase running during the build time of the package.                                                                                  | No, since the tool has no support for executing `passthru.tests`. [^1]                                                                                                                                      |
+| Package breakage awareness             | Loud and clear as soon as the hook is reached, since it is a phase running during the build time of the package.                                              | Requires specific command to be noticed, e.g. `nix-build -A pkg.tests.version`, since it is a derivation dependent on the package                                                                           |
+| Transitive package breakage            | Never ever goes unnoticed by `nixpkgs-review`, since the tool executes the transitive dependencies.                                                           | Since human beings are prone to forget the duty of running passthru tests and `nixpkgs-review` has no support for running it, it certainly goes unnoticed.                                                  |
```

Comment:
Haha this:

> human beings are prone to forget the duty of running passthru tests

Is a very blaming phrasing.. I don't think there is a consensus that people should run `passthru.tests` of all packages they use on their system regularly or whatever. It is not exactly clear in which context it would be correct to blame someone about such a breakage. My opinion is that no one should be blamed for not doing something that our tooling doesn't allow them to easily do.

Reply:
I will remove references to less-than-omniscient things in the next iteration.